apps/extension/src/routes/service-worker/extension/sync.ts

```diff
@@ -1,9 +1,13 @@
-import { SwMessage, SwMessageHandler } from './types';
 import { services } from '../../../service-worker';
+import { SwMessage, SwMessageHandler } from './types';
 
-export type SyncBlocksMessage = SwMessage<'SYNC_BLOCKS', undefined, Promise<void>>;
+export type SyncBlocksMessage = SwMessage<'SYNC_BLOCKS' | 'CLEAR_CACHE', undefined, Promise<void>>;
```

Comment:
Messages should not share definitions like that. Should create a new one for `ClearCacheMessage`.